tests/pytests/pkg/download/test_pkg_download.py

```diff
@@ -256,6 +256,9 @@ def setup_redhat_family(
 ):
     arch = os.environ.get("SALT_REPO_ARCH") or "x86_64"
 
+    if os_name == "photon":
+        os_version = f"{os_version}.0"
```

Comment:
Wondering what happens when '.1' comes along

Reply:
It doesn't, never has.

Reply:
But @felippeb knows more about that information.

Reply:
The photon team has indicated that they have never used that versioning. they put a .X in place if they ever wanted to, but functionally never have.